lib/util.js

```diff
@@ -301,10 +301,10 @@ function ensureDebugIsInitialized() {
 
 
 function inspectPromise(p) {
-  ensureDebugIsInitialized();
   // Only create a mirror if the object is a Promise.
```

Comment:
Where to? Wouldn't the best place for the comment about checking whether it's a promise be right before the line that checks if it's a promise?